pages/api/updateContact.ts

```diff
@@ -26,7 +26,11 @@ export default async function handler (
       if (user !== null && user !== undefined) {
         const contactToAdd = fillContact(req)
         if (await contactManager.modifyContact(user.id, contactToAdd, req.body.index)) {
-          res.status(200).end()
+          res.status(201).json({
```

Comment:
I can't think rn, but doesn't this just send the last contact for the user back?